bluebutton-data-pipeline-rif-extract/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/RifFilesProcessor.java

```diff
@@ -290,7 +290,27 @@ private static BeneficiaryRow buildBeneficiaryEvent(CSVRecord csvRecord) {
 		beneficiaryRow.stateCode = csvRecord.get(BeneficiaryRow.Column.STATE_CODE.ordinal());
 		beneficiaryRow.countyCode = csvRecord.get(BeneficiaryRow.Column.BENE_COUNTY_CD.ordinal());
 		beneficiaryRow.postalCode = csvRecord.get(BeneficiaryRow.Column.BENE_ZIP_CD.ordinal());
-		// TODO finish mapping the rest of the columns
+		beneficiaryRow.birthDate = LocalDate.parse(csvRecord.get(BeneficiaryRow.Column.BENE_BIRTH_DT.ordinal()),
```

Comment:
I haven't updated any of my code either to take advantage of it yet, but now that the RIF files have a header row, we can (and **should**) stop using `.ordinal()` in all of this code. That will make the parsing more robust to future column order changes.